src/components/NewPost/NewPost.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import users from '../../api/users';
+
+type Props = {
+  onAdd: (Todo: Todo) => void;
+};
+
+export const NewPost: React.FC<Props> = ({ onAdd }) => {
+  const [userId, setUserId] = useState(0);
+  const [title, setTitle] = useState<string>('');
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
```
it's redundant to specify primitives